view/js/obj.js

```diff
@@ -108,12 +108,12 @@ wab.Obj.prototype.save = function() {
             if (4 == h.readyState) {
                 if (200 == h.status) {
                     if (0 == o.ref) { // create
-                        o.ref = h.response.body.ref
+                        o.ref = h.response.ref
                         o.delete_button.style.visibility = 'visible';
                         o.save_button.removeChild(o.save_button.firstChild);
                         o.save_button.appendChild(document.createTextNode('Update'));
                     } else { // update
-                        var updated = h.response.body.updated;
+                        var updated = h.response.updated;
```

Comment:
will this change mean things won't work as they used to with the OpO runner?

Reply:
It would If I hadn't updated opod. :-)